### ISSUE TITLE
官网挂了

### ISSUE THREAD

**Issue by r372801950** [winston779/xrelay#1]
你官网挂了

**Comment by norman93sun** [comment]
404

**Comment by fengshanjian** [comment]
跑路了？

**Comment by liandao0815** [comment]
跑路了

**Comment by fine-wang** [comment]
how time deal？

**Comment by jiajihao** [comment]
真跑路了吗，官网都连不上了

**Comment by MingsQaQ** [comment]
RIP

**Comment by pessiTra** [comment]
官网、app全没反应，这也太逆天了

**Comment by dujie-js** [comment]
我tm刚充一年的会员

**Comment by yigenhuobahh** [comment]
草拟吗

**Comment by ddslo** [comment]
真跑了？

**Comment by sldsg** [comment]
官网已更改：https://crelay.net/，所有用户补偿7天。

**Comment by yigenhuobahh** [comment]
新官网无响应

**Comment by sldsg** [comment]
> 新官网无响应

https://crelay.net/

**Comment by yigenhuobahh** [comment]
![image](https://github.com/winston779/xrelay/assets/121288397/3156e282-be4e-4e3a-b076-fe4c6a088da2)
如图，官网有bug但线路可正常链接

**Comment by WentaoLu815** [comment]
> ![image](https://github.com/winston779/xrelay/assets/121288397/3156e282-be4e-4e3a-b076-fe4c6a088da2)
> 
> 如图，官网有bug但线路可正常链接

兄弟有解决办法吗

**Comment by yigenhuobahh** [comment]
> > ![image](https://private-user-images.githubusercontent.com/121288397/334826587-3156e282-be4e-4e3a-b076-fe4c6a088da2.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTY5OTQzNDUsIm5iZiI6MTcxNjk5NDA0NSwicGF0aCI6Ii8xMjEyODgzOTcvMzM0ODI2NTg3LTMxNTZlMjgyLWJlNGUtNGUzYS1iMDc2LWZlNGM2YTA4OGRhMi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNTI5JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDUyOVQxNDQ3MjVaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT03YzEzN2NmYmEyNmQ4YjU3NGNlMTIxYmViM2U2NTFiYWVkZTZhYmU5NDBjYTZkMTEzYjIxNTBjZGIwN2NhNjkwJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.NJqryuZbhVLWLMx0qfEXpDKSnqlOG9boqorBEcznyj8)
> > 如图，官网有bug但线路可正常链接
> 
> 兄弟有解决办法吗

https://[crelay](https://crelay.net/#/login).net/#/login
从这里登录
https://[crelay](https://crelay.net/#/knowledge).net/#/knowledge
帮助中心可以正常访问，之后即可使用线路

**Comment by Collapsar-X** [comment]
> > > ![image](https://private-user-images.githubusercontent.com/121288397/334826587-3156e282-be4e-4e3a-b076-fe4c6a088da2.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTY5OTQzNDUsIm5iZiI6MTcxNjk5NDA0NSwicGF0aCI6Ii8xMjEyODgzOTcvMzM0ODI2NTg3LTMxNTZlMjgyLWJlNGUtNGUzYS1iMDc2LWZlNGM2YTA4OGRhMi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNTI5JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDUyOVQxNDQ3MjVaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT03YzEzN2NmYmEyNmQ4YjU3NGNlMTIxYmViM2U2NTFiYWVkZTZhYmU5NDBjYTZkMTEzYjIxNTBjZGIwN2NhNjkwJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.NJqryuZbhVLWLMx0qfEXpDKSnqlOG9boqorBEcznyj8)
> > > 如图，官网有bug但线路可正常链接
> > 
> > 
> > 兄弟有解决办法吗
> 
> https://[crelay](https://crelay.net/#/login).net/#/login 从这里登录 https://[crelay](https://crelay.net/#/knowledge).net/#/knowledge 帮助中心可以正常访问，之后即可使用线路
However, when I click on the link you provided, the page is not responsive and the node is in an unavailable state and the subscription information update fails, how can I resolve this situation?

**Comment by yigenhuobahh** [comment]
目前已正常，请重试

**Comment by Collapsar-X** [comment]
> 目前已正常，请重试

Yes, I just re-tried it and it does work, thanks for the answer!
